README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
 
 ### Supported GPUs
 
-#### Radeon Pro V620 and W6800 Workstation GPUs
+Support can be assumed for current server accelerators GPUs intended for enterprise customers.
```

Comment:
"current server accelerators and GPUs"?

Reply:
Good catch. Yes, I was missing an "and" here. I added it in my latest commit.